app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -578,9 +578,13 @@ private void decideButtonVisibilities() {
      *
      */
     private void addActionToTitle() {
+        title.setOnLongClickListener(view -> {
+                Utils.copy("place", title.getText().toString(), getContext());
+                Toast.makeText(getContext(), "Text copied to clipboard", Toast.LENGTH_SHORT).show();
```

Comment:
We should not be putting any hard coded strings.

Reply:
Thanks for pointing it out, I have made the change. :)